tests/load-tests/clear.sh

```diff
@@ -1,10 +1,12 @@
-REPO_LIST=$(curl -s https://api.github.com/users/app-studio-test/repos\?per_page\=1000 | jq -r '.[]|select(.name | startswith("testuser"))' | jq --raw-output '.name')
+#!/bin/bash -x
+USER_PREFIX=${1:-testuser}
+REPO_LIST=$(curl -s https://api.github.com/users/${MY_GITHUB_ORG}/repos\?per_page\=1000 | jq -r '.[]|select(.name | startswith("'${USER_PREFIX}'"))' | jq --raw-output '.name')
```

Comment:
Paging in github api is limited to 100 per page, not 1000 - need to implement paging to be able to list all the repos.

Reply:
Cherry-picking @jhutar's commit (https://github.com/redhat-appstudio/e2e-tests/pull/403) to solve this.

Reply:
Oh, sent email. Anyway see https://github.com/redhat-appstudio/e2e-tests/pull/403/commits/75323666739b22e75173d5cda3ea3cafc076b1dd

Reply:
Hi @pmacik 

You can fix it as follows:

REPO_LIST=$(curl -s https://api.github.com/users/app-studio-test/repos\?per_page\=100 | jq -r '.[]|select(.name | startswith("testuser"))' | jq --raw-output '.name')

while [ "$REPO_LIST" != "" ]; do
  for REPO in $REPO_LIST
  do
    curl \
      -X DELETE \
      -H "Accept: application/vnd.github.v3+json" \
      -H "Authorization: token $GITHUB_TOKEN" \
      https://api.github.com/repos/app-studio-test/${REPO}
  done

  # Check if there is a next page of data
  next_page_url=$(curl -s -I https://api.github.com/users/app-studio-test/repos\?per_page\=100 | awk '/^Link:/ { print $2 }')
  if [ "$next_page_url" != "" ]; then
    REPO_LIST=$(curl -s "$next_page_url" | jq -r '.[]|select(.name | startswith("testuser"))' | jq --raw-output '.name')
  else
    REPO_LIST=""
  fi
done

Reply:
@jhutar thx, I was going to cherry-pick your commit, but then I realized, that it does not actually perform paging (the `page` parameter is never set to the page number in your script). So, I implemented it myself (https://github.com/redhat-appstudio/e2e-tests/pull/396/files#diff-3ad491e230a37fdeb948f6905d647fe2506a8e4360498a82bdd1593f6f45574b).

Reply:
Thx @naftalysh for the suggestion, however there are couple of other things that I wanted to fix (such as fixing github API URL - we need to use `.../orgs/...` instead of `.../users/...`, un-hard-coding the `app-studio-test` org, un-hard-coding name of the test user prefix, simplifying the use of `jq` in getting the repo list, unifying `curl` arguments to use authenticated calls and to keep it at one place only etc.). So I decided to implement it myslef, and I did before you suggested similar solution ;)

Reply:
Just a note: because we delete first 100 repos, we do not need to do paging when listing second 100 of repos because we will just get different 100 hosts. Anyway thank you for implementing it!